crates/networking/p2p/src/network.rs

```diff
@@ -153,15 +151,11 @@ impl Network {
             }
         }
 
-        let bootnodes = Bootnodes::new();
-
-        for bootnode in bootnodes.bootnodes {
-            if let Some(ipv4) = bootnode.ip4() {
+        for bootnode in config.boot_nodes_enr.clone() {
```

Comment:
```suggestion
        for bootnode in &config.boot_nodes_enr {
```

we shouldn't need a clone